app/models.py

```diff
@@ -5,3 +5,149 @@
 Author: huxuan <i(at)huxuan.org>
 Description: Models for app.
 """
+from app import db
+
+
+class User(db.Model):
+    __tablename__ = 'userinfo'
+
+    # new segments
+    uid = db.Column(db.Integer, primary_key=True)
+    birthday = db.Column(db.DateTime)
+    student_id = db.Column(db.String(16))
+    name = db.Column(db.String(16))
+    tel = db.Column(db.String(16))
+
+    # existing segments
+    username = db.Column(db.String(32), unique=True)
+    password = db.Column(db.String(32), nullable=False)
+    # token temporarily not included
+    # tokentime temporarily not included
+    gender = db.Column(db.Boolean, nullable=False)  # was `sex`
+    icon = db.Column(db.Text)
+    intro = db.Column(db.Text)
+    sig1 = db.Column(db.Text)
+    sig2 = db.Column(db.Text)
+    sig3 = db.Column(db.Text)
+    hobby = db.Column(db.Text)
+    qq = db.Column(db.String(16))
+    mail = db.Column(db.String(64))
+    province = db.Column(db.String(8))  # was `place`
+    registration_date = db.Column(db.DateTime)  # was `regdate`
+    last_login_date = db.Column(db.DateTime)  # was `lastdate`
+    last_login_ip = db.Column(db.DateTime)  # was `lastip`
+    star = db.Column(db.Integer)
+    # score temporarily not included
+    post_num = db.Column(db.Integer, unsigned=True)  # was `post`
+    reply_num = db.Column(db.Integer, unsigned=True)  # was `reply`
+    water_num = db.Column(db.Integer, unsigned=True)  # was `water`
+    sign_num = db.Column(db.Integer, unsigned=True)  # was `sign`
+    # rights temporarily not included
+    # newmsy temporarily not included
+    # extr temporarily not included
+    last_post_time = db.Column(db.DateTime)  # was `lastpost`
+    current_board = db.Column(db.Integer)  # was `nowboard`
+    user_agent = db.Column(db.String(128))  # was `onlinetype`
+    # logininfo temporarily not included
+    # code temporarily not included
+    # other2 3 4 5 6 temporarily not included
+
+
+class Board(db.Model):
+    __tablename__ = 'boards'
+
+    bid = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(32), unique=True)  # was `bbstitle`
+    invisible = db.Column(db.Boolean, default=False)  # was `hide`
+
```

Comment:
To make the code more easily to read. I prefer to organize the order as something like `User Group Board Thread Post Comment Message Notification`